Roslyn.sln

```diff
@@ -322,6 +316,24 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MSBuildTask.Desktop", "src\
 EndProject
 Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "MSBuildTask.Shared", "src\Compilers\Core\MSBuildTask\Shared\MSBuildTask.Shared.shproj", "{D87F0E46-DD1B-46EA-8425-9E185D9B602E}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "PerfTest", "PerfTest", "{7B5941C4-BF81-4A9D-91A7-DE524058A47B}"
+	ProjectSection(SolutionItems) = preProject
+		src\Test\Perf\runner.csx = src\Test\Perf\runner.csx
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "helloworld", "helloworld", "{002DD150-5946-4AB6-9B0F-6A3CAAA782E0}"
```

Comment:
We probably don't.  I added them because someone asked how to open them in visual studio and I didn't have an answer to that.

Is there a better way to make that workflow happen?